pkg/api/rest/rest.go

```diff
@@ -154,19 +154,15 @@ type CollectionDeleter interface {
 
 // Creater is an object that can create an instance of a RESTful object.
 type Creater interface {
-	// New returns an empty object that can be used with Create after request data has been put into it.
-	// This object must be a pointer type for use with Codec.DecodeInto([]byte, runtime.Object)
-	New() runtime.Object
+	Storage
```

Comment:
Sorry, I didn't think it through last time. The resthandler [calls](https://github.com/kubernetes/kubernetes/blob/master/pkg/apiserver/resthandler.go#L377) the `New()` method of `Creater` etc. Compared to embedding the Storage interface, explicitly defining `New()` as an interface method looks more readable to me. Hence, I don't feel we need to change this file.